bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/AmazonEchoControlBindingConstants.java

```diff
@@ -1,119 +1,119 @@
-    public static final String CHANNEL_AMAZON_MUSIC = "amazonMusic";
-    public static final String CHANNEL_AMAZON_MUSIC_PLAY_LIST_ID = "amazonMusicPlayListId";
-    public static final String CHANNEL_TEXT_TO_SPEECH = "textToSpeech";
-    public static final String CHANNEL_TEXT_TO_SPEECH_VOLUME = "textToSpeechVolume";
-    public static final String CHANNEL_REMIND = "remind";
-    public static final String CHANNEL_PLAY_ALARM_SOUND = "playAlarmSound";
-    public static final String CHANNEL_START_ROUTINE = "startRoutine";
-    public static final String CHANNEL_MUSIC_PROVIDER_ID = "musicProviderId";
-    public static final String CHANNEL_PLAY_MUSIC_VOICE_COMMAND = "playMusicVoiceCommand";
-    public static final String CHANNEL_START_COMMAND = "startCommand";
-    public static final String CHANNEL_LAST_VOICE_COMMAND = "lastVoiceCommand";
-    public static final String CHANNEL_MEDIA_PROGRESS = "mediaProgress";
-    public static final String CHANNEL_MEDIA_LENGTH = "mediaLength";
-    public static final String CHANNEL_MEDIA_PROGRESS_TIME = "mediaProgressTime";
-    public static final String CHANNEL_ASCENDING_ALARM = "ascendingAlarm";
-    public static final String CHANNEL_NOTIFICATION_VOLUME = "notificationVolume";
-    public static final String CHANNEL_NEXT_REMINDER = "nextReminder";
-    public static final String CHANNEL_NEXT_ALARM = "nextAlarm";
-    public static final String CHANNEL_NEXT_MUSIC_ALARM = "nextMusicAlarm";
-    public static final String CHANNEL_NEXT_TIMER = "nextTimer";
-
-    public static final String CHANNEL_SAVE = "save";
-    public static final String CHANNEL_ACTIVE = "active";
-    public static final String CHANNEL_PLAY_ON_DEVICE = "playOnDevice";
-
-    // List of channel Type UIDs
-    public static final ChannelTypeUID CHANNEL_TYPE_BLUETHOOTH_MAC = new ChannelTypeUID(BINDING_ID, "bluetoothMAC");
-    public static final ChannelTypeUID CHANNEL_TYPE_AMAZON_MUSIC_PLAY_LIST_ID = new ChannelTypeUID(BINDING_ID,
-            "amazonMusicPlayListId");
-    public static final ChannelTypeUID CHANNEL_TYPE_PLAY_ALARM_SOUND = new ChannelTypeUID(BINDING_ID, "playAlarmSound");
-    public static final ChannelTypeUID CHANNEL_TYPE_CHANNEL_PLAY_ON_DEVICE = new ChannelTypeUID(BINDING_ID,
-            "playOnDevice");
-    public static final ChannelTypeUID CHANNEL_TYPE_MUSIC_PROVIDER_ID = new ChannelTypeUID(BINDING_ID,
-            "musicProviderId");
-    public static final ChannelTypeUID CHANNEL_TYPE_START_COMMAND = new ChannelTypeUID(BINDING_ID, "startCommand");
-
-    // List of all Properties
-    public static final String DEVICE_PROPERTY_SERIAL_NUMBER = "serialNumber";
-    public static final String DEVICE_PROPERTY_FAMILY = "deviceFamily";
-    public static final String DEVICE_PROPERTY_FLASH_BRIEFING_PROFILE = "configurationJson";
-    public static final String DEVICE_PROPERTY_ID = "id";
-
-    // Other
-    public static final String FLASH_BRIEFING_COMMAND_PREFIX = "FlashBriefing.";
-}
+/**
```

Comment:
Can you check what happened here? All lines are changed, that usually is either an encoding or line-ending issue

Reply:
I use a netbeans plugin for line endings to prevent windows line endings. I also installed this plugin on my mac preventively anyway it should not be necessary on a unix system. Never had any issue with it but maybe it produces line ending problems. I never changed the files you listed, so this is strange. But i have seen them in the commit and ignored it. In the future i will take a closer look before i commit. I reset all three files like you suggested.

Reply:
It seems that didn't work. If you remove the branch protection rule under https://github.com/Trinitus01/openhab-addons/settings/branches and check `allow edits by maintainers` I can fix that myself.

Reply:
I removed the branch protection. Allow edits by maintainers is check for this pull request.